index.d.ts

```diff
@@ -98,6 +99,15 @@ export interface Space {
     toPlainObject(): Space;
 }
 
+export interface Locales {
+    sys: Sys;
+    total: number;
+    skip: number;
+    limit: number;
+    items: Array<string>;
```

Comment:
Shouldn't this return a `Locale` type - which currently does not exist - or `any`?

I think the Locale type would look something like

```
interface Locale {
  code: string
  name: string
  default: boolean
  fallbackCode: string | null
  sys: {
    id: string
    type: 'Locale'
    version: number
  }
}
```

Reply:
This should be `ContentfulCollection<Locale>` Locales should be removed and as Tim mentioned we should add Locale interface

```js
interface Locale {
  code: string
  name: string
  default: boolean
  fallbackCode: string | null
  sys: {
    id: string
    type: 'Locale'
    version: number
  }
}
```

Reply:
Thanks @Khaledgarbaya and @TimBeyer for the feedback and directions.
I addressed your feedback and pushed the changes to this PR.